ios/findAR.xcodeproj/project.pbxproj

```diff
@@ -105,17 +113,17 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXFileReference section */
-		008F07F21AC5B25A0029DE68 /* main.jsbundle */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text; name = main.jsbundle; path = main.jsbundle; sourceTree = "<group>"; };
```

Comment:
@bignell Can you verify if this is necessary? It works when I test it on my phone, so I think it's ok.